src/utilcode/pedecoder.cpp

```diff
@@ -2481,14 +2481,34 @@ PTR_CVOID PEDecoder::GetNativeManifestMetadata(COUNT_T *pSize) const
     CONTRACT(PTR_CVOID)
     {
         INSTANCE_CHECK;
-        PRECONDITION(CheckNativeHeader());
+        PRECONDITION(HasReadyToRunHeader() || CheckNativeHeader());
         POSTCONDITION(CheckPointer(RETVAL, NULL_OK)); // TBD - may not store metadata for IJW
         NOTHROW;
         GC_NOTRIGGER;
     }
     CONTRACT_END;
+    
+    IMAGE_DATA_DIRECTORY *pDir;
+    if (HasReadyToRunHeader() && IsLargeVersionBubbleEnabled())
```

Comment:
`IsLargeVersionBubbleEnabled` should be used during compile time only. It is not valid at runtime (always returns false). This piece of code runs at runtime. I suspect that this IsLargeVersionBubbleEnabled effectively disabled loading of large version bubble assemblies.

Reply:
Absolutely correct. Thanks for the spot.